### PR TITLE
Fix the build and start script

### PR DESCRIPTION
**Pull request by janderson2**
Bring the build and start script inline with standards.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,28 +1,39 @@
...
-VERSION=1.0.0
-TESTS?=./...
-CHS_ENV_HOME?=$(HOME)/.chs_env
-CHS_ENVS=$(CHS_ENV_HOME)/global_env
-SOURCE_ENV=for chs_env in $(CHS_ENVS); do test -f $$chs_env && . $$chs_env; done
+artifact_name       := document-render-tiff2pdf
+commit              := $(shell git rev-parse --short HEAD)
+tag                 := $(shell git tag -l 'v*-rc*' --points-at HEAD)
+version             := $(shell if [[ -n "$(tag)" ]]; then echo $(tag) | sed 's/^v//'; else echo $(commit); fi)
+artifactory_publish := $(shell if [[ -n "$(tag)" ]]; then echo release; else echo dev; fi)
```
Comment (Jata2k) [inline Makefile]: Can this come out?

File: pom.xml [hunk 2]
```diff
@@ -1,162 +1,96 @@
...
-                </execution>
-            </executions>
-        </plugin>
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>uk.gov.companieshouse</groupId>
+    <artifactId>document-renderer-tiff2pdf</artifactId>
+    <version>3103253</version>
```
Comment (Jata2k) [inline pom.xml]: Should this be unversioned?

File: Makefile (hunk 1 above)
Comment (janderson2) [reply Makefile]: It can indeed.

File: pom.xml [hunk 3]
```diff
@@ -1,162 +1,96 @@
...
+
+    <build>
+        <finalName>document-render-tiff2pdf</finalName>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>${maven-compiler-plugin.version}</version>
+                <configuration>
+                    <source>1.8</source>
```
Comment (Jata2k) [inline pom.xml]: Put this in properties section?

**Review by Jata2k — APPROVED** [review]

**Pull request merged**